.travis.yml

```diff
@@ -2,20 +2,20 @@ language: python
 dist: xenial
 
 python:
-  - "3.6"
   - "3.7"
 
 addons:
   apt:
     packages:
       - language-pack-nl
 
-install:
-  - pip install --upgrade pip setuptools
-  - pip install -r requirements.txt
-  - pip install -r dev-requirements.txt
-  - pip install coveralls
+before_install:
+  - pip install --upgrade setuptools
+  - pip install poetry
 
+install:
+  - poetry install
+  - poetry add coveralls
```

Comment:
Why not add this to the dev requirements?

Reply:
Same reason why it previously wasn't: it is not a dev dependency

Reply:
This should've been `pip install` I think – a one-off installation of an executable tool shouldn't invoke poetry's quite expensive dependency locking framework.